src/javaScript/cart.js

```diff
@@ -1,2 +1,80 @@
 import '../main.min.css'
 import '../cart.scss'
+
+function getCartItems () {
+  return JSON.parse(localStorage.getItem('cart')) || []
+}
+
+export function displayCartItems () {
+  const cartItems = getCartItems()
+  const productItemCart = document.getElementById('product-item-cart')
+  const priceItemCart = document.getElementById('price-item-cart')
+  const quantityItemCart = document.getElementById('quantity-item-cart')
+  const subtotalItemCart = document.getElementById('subtotal-item-cart')
+
+  if (
+    !productItemCart ||
+    !priceItemCart ||
+    !quantityItemCart ||
+    !subtotalItemCart
+  ) { return }
+
+  productItemCart.innerHTML = 'Product'
+  priceItemCart.innerHTML = 'Price'
+  quantityItemCart.innerHTML = 'Quantity'
```

Comment:
They already have this setup from the html structure. Not clear why this is needed as well